ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh

```
@@ -73,7 +73,14 @@ then
-openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-DISABLED_MONITOR_TESTS="apiserver-new-disruption-invariant,disruption-summary-serializer,incluster-disruption-serializer,pod-network-avalibility"
-openshift-tests run-monitor ${MONITOR_ARGS:-} --artifact-dir $STORE_PATH --disable-monitor=${DISABLED_MONITOR_TESTS} > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
+EVENTS_FLAG=""
+if [[ "${RESOURCE_EVENT_COLLECTION_ENABLED:-false}" == "true" ]]; then
+  EVENTS_FLAG="--enable-events"
+fi
+openshift-tests run-resourcewatch ${EVENTS_FLAG} > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
+
+if [[ "${RESOURCE_MONITOR_ENABLED:-false}" == "true" ]]; then
+  DISABLED_MONITOR_TESTS="apiserver-new-disruption-invariant,disruption-summary-serializer,incluster-disruption-serializer,pod-network-avalibility"
+  openshift-tests run-monitor ${MONITOR_ARGS:-} --artifact-dir $STORE_PATH --disable-monitor=${DISABLED_MONITOR_TESTS} > "${ARTIFACT_DIR}/run-monitor.log" 2>&1 &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)